lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3609,7 +3609,18 @@ namespace {
           isa<clang::FunctionDecl>(decl)
               ? cast<clang::FunctionDecl>(decl)->getReturnType()
               : cast<clang::ObjCMethodDecl>(decl)->getReturnType();
-      if (isForeignReferenceTypeWithoutImmortalAttrs(retType)) {
+      clang::QualType pointeeType = retType;
```

Comment:
This was the only use of `isForeignReferenceTypeWithoutImmortalAttrs`. I think we could have fixed that function. Now I think it is unused so let's remove it.